api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -35,20 +56,32 @@
 
   @Autowired
   public ProfileService(
```

Comment:
We should probably make a separate ProfileService interface and make this a ProfileServiceImpl.